test/js/specs/happy.js

```diff
@@ -12,7 +13,9 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
   describe('welcome screen', function () {
-    const copy = welcome.copy()
+
+    const copy = locale("en")
```

Comment:
If you don't pass anything it will default to "en". I think you can leave "en" for now as it will make it clearer that we are only testing "en" for now.